packages/@aws-cdk/pipelines/lib/lambda/index.py

```diff
@@ -0,0 +1,35 @@
+import boto3
+import json
+
+cp = boto3.client('codepipeline')
+
+def handler(event, context):
+  pipeline_name = event['PipelineName']
+  stage_name = event['StageName']
+  action_name = stage_name + 'ManualApproval'
```

Comment:
Weelll... it's a bit dangerous that we duplicate the naming conventions in two places. Couldn't we just pass this name through?

Reply:
Yeaa we could. Wanted to save some space in the function call but this works.